packages/ui-kit/src/components/Checkbox/Checkbox.tsx

```diff
@@ -0,0 +1,54 @@
+import { FC, ReactNode, InputHTMLAttributes } from "react";
+import clsx from "clsx";
+import { nanoid } from "nanoid";
+
+import styles from "./Checkbox.module.css";
+import { CheckIcon } from "../icons";
+import { Label } from "../Label";
+
+export interface CheckboxProps
+  extends Omit<InputHTMLAttributes<HTMLInputElement>, "className"> {
+  label?: string | ReactNode;
```

Comment:
How are we set up on accessibility here? @jwm0 any ideas? Should we enforce this or not? (since it's a lib to be used only in Saleor projects as far as I understand, so maybe we could?)

Reply:
UI Kit might be use outside of company I guess. 
It shouldn't impose dataset o arias. It's definitely good practice, but is it force? 🤔

Reply:
I'd not enforce it, right now it's internal but it's definitely intended to be used by a broader audience